tests/nativeapp/test_manager.py

```diff
@@ -1701,61 +1701,3 @@ def test_stream_events(mock_execute, mock_account_event_table, temp_dir, mock_cu
         pass
     else:
         pytest.fail("stream_events didn't end when receiving a KeyboardInterrupt")
-
-
-@mock.patch.object(NativeAppManager, "validate")
-@mock.patch.object(NativeAppManager, "execute_package_post_deploy_hooks")
-@mock.patch.object(NativeAppManager, "sync_deploy_root_with_stage")
-@mock.patch.object(NativeAppManager, "_apply_package_scripts")
-@mock.patch.object(NativeAppManager, "create_app_package")
-@mock.patch.object(NativeAppManager, "use_role")
-@mock.patch.object(NativeAppManager, "use_package_warehouse")
-def test_deploy_with_package_post_deploy_hook(
```

Comment:
we test deploy in the ApplicationPackageEntity tests, which now covers all PDF versions